static/src/hbs/_scripts.hbs

```diff
@@ -1 +1,2 @@
-<script src='{{ assetPath "js/site.js" }}'></script>
+{% load staticfiles %}
```

Comment:
So like this?

    {% load staticfiles %}<script src='{% static "{{ assetPath "js/site.js" }}" %}'></script>

Do you just want it to be a one-liner for some reason?  I've typically seen Django template loads on separate lines.